prow/ANNOUNCEMENTS.md

```diff
@@ -3,6 +3,7 @@
 ## New features
 
 New features added to each component:
+  - *April 14, 2021* Deck remove default value for `--cookie-secret-file`.
```

Comment:
This should go under the breaking changes section, not the new features section.

Reply:
oops, thanks Cole!